cilium-dbg/cmd/bpf_multicast_groups.go

```diff
@@ -0,0 +1,152 @@
+	Short: "Manage the multicast groups.",
+}
+
+func init() {
+	BpfMcastGroupCmd.AddCommand(MulticastGroupListCmd)
+	command.AddOutputOption(MulticastGroupListCmd)
+	BpfMcastGroupCmd.AddCommand(MulticastGroupAddCmd)
+	BpfMcastGroupCmd.AddCommand(MulticastGroupDelCmd)
+}
+
+var MulticastGroupListCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Short:   "List the multicast groups.",
+	Long:    "List the multicast groups configured on the node.",
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf multicast group list")
+
+		groupV4Map, err := maps_multicast.OpenGroupV4OuterMap(maps_multicast.GroupOuter4MapName)
+		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				fmt.Fprintln(os.Stderr, "Cannot find multicast bpf maps")
+				return
+			}
+
+			Fatalf("Cannot open multicast bpf maps: %s", err)
+		}
+
+		groups, err := groupV4Map.List()
+		if err != nil {
+			Fatalf("Error listing multicast groups: %s", err)
+		}
+
+		if command.OutputOption() {
+			if err := command.PrintOutput(groups); err != nil {
+				Fatalf("error getting output of map in %s: %s\n", command.OutputOptionString(), err)
+			}
+			return
+		}
+		printGroupList(groups)
+	},
+}
+
+func printGroupList(groups []netip.Addr) {
+	// sort groups by address
+	sort.Slice(groups, func(i, j int) bool {
+		return groups[i].Compare(groups[j]) < 0
+	})
+
+	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
+	fmt.Fprintln(w, "Group Address")
+	for _, group := range groups {
+		fmt.Fprintf(w, "%s\n", group.String())
+	}
+	w.Flush()
+}
+
+var MulticastGroupAddCmd = &cobra.Command{
+	Use:   "add <group>",
+	Short: "Add a multicast group.",
+	Long:  "Add a multicast group to the node.",
+	Example: `	Following command adds group 229.0.0.1 to BPF multicast map of the node:
+cilium-dbg bpf multicast group add 229.0.0.1`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium-dbg bpf multicast group add")
+
+		if len(args) != 1 {
+			Fatalf("Invalid number of arguments, expected 1, got %d", len(args))
+		}
+
+		group, err := netip.ParseAddr(args[0])
+		if err != nil {
+			Fatalf("Invalid IP address format: %s", err)
+		}
+
+		// check group is multicast and ipv4 address
+		if !group.Is4() || !group.IsMulticast() {
+			Fatalf("Invalid multicast group: %s", group.String())
+		}
+
+		groupV4Map, err := maps_multicast.OpenGroupV4OuterMap(maps_multicast.GroupOuter4MapName)
+		if err != nil {
+			Fatalf("Cannot open multicast bpf maps: %s", err)
+		}
+
+		err = groupV4Map.Insert(group)
+		if err != nil {
+			Fatalf("Error adding multicast group: %s", err)
+		}
+	},
+}
+
+var MulticastGroupDelCmd = &cobra.Command{
+	Use:     "delete <group>",
+	Aliases: []string{"del"},
+	Short:   "Delete a multicast group.",
+	Long:    "Delete a multicast group from the node.",
+	Example: `	Following command deletes group 229.0.0.1 from BPF multicast map of the node:
+cilium-dbg bpf multicast group delete 229.0.0.1`,
```

Comment:
```suggestion
	Example: `The ollowing command deletes group 229.0.0.1 from BPF multicast map of the node:

    cilium-dbg bpf multicast group delete 229.0.0.1`,
```

Same as for the add command, currently it renders a bit weird.